site/content/en/docs/concepts/_index.md

```diff
@@ -8,11 +8,11 @@ no_list: true
 ---
 
 # LeaderWorkerSet (LWS)
-An LWS creates a group of pods based on two different templates (the leader template and the worker template), and controls their lifecycle. 
+An LWS creates a group of pods based on two different templates (the leader template and the worker template), and controls their lifecycle.
 
 ## Conceptual Diagram
 
-![LWS diagram](https://github.com/user-attachments/assets/5c413f5d-8c34-488f-b92e-339273555a7d)
+![LWS diagram](../../images/concept.png)
```

Comment:
Does /images/concept.png or ../images/concept.png is better :-)

Reply:
From the website's perspective, it's going to be https://lws.sigs.k8s.io//images/concept.png, which I am fine with that. Kueue (https://github.com/kubernetes-sigs/kueue/tree/main/site/static/images) also stores its website images in `site/static/images`